// Operations that can be performed on injectables.

syntax = "proto3";

package inhumate.rti.proto;
option csharp_namespace = "Inhumate.RTI.Proto";

import "google/protobuf/empty.proto";

message InjectableOperation {

    oneof which {
        google.protobuf.Empty request_update = 1;
    }

}
